lib/lotus/helpers/html_helper/empty_html_node.rb

```diff
@@ -48,7 +48,11 @@ def attributes
           result = [nil]
 
           @attributes.each do |name, value|
-            result << %(#{ name }="#{ value }")
+            if %('required' 'readonly' 'disabled' 'autofocus').include? name.to_s
```

Comment:
According to HTML5 spec:

```
When a control has no value but has a required attribute (input required, select required, textarea required), or, in the case of an element in a radio button group, any of the other elements in the group has a required attribute.
```

you can see that constraints are available to limited number of tags, for the sake of this PR, let's stick to adding support for `required` constraint for `input`, `select`, `textarea` and `radio button`